spec/table_spec.rb

```diff
@@ -2792,6 +2790,24 @@
         end
       end
     end
+
+    context "when `except` is passed" do
+      it 'resizes only the other columns' do
+        # Check that an Array argument works
+        table = Tabulo::Table.new(%w(hello hi there), :to_s, :length, :upcase)
+        expect { table.pack(except: [:to_s, :length]) }
+          .to change(table.column_registry[:to_s], :width).by(0)
+          .and change(table.column_registry[:length], :width).by(0)
+          .and change(table.column_registry[:upcase], :width).to(6)
+
+        # Check that a single argument works
+        table = Tabulo::Table.new(%w(hello hi there), :to_s, :length, :upcase)
+        expect { table.pack(except: :length) }
+          .to change(table.column_registry[:to_s], :width).to(5)
+          .and change(table.column_registry[:length], :width).by(0)
+          .and change(table.column_registry[:upcase], :width).to(6)
+      end
+    end
```

Comment:
These tests look pretty good. However for this library, I like to test the various parameters, methods etc. against what I expect the end user to actually see, i.e. the rendered table. See the existing tests for precedent.

I know it can be relatively laborious to construct such tests, however, it gives me a level of confidence in correctness of the behaviour, that is greater than what I feel can be provided with internally focused unit tests alone.

Up to you if you want to write the "rendered table" test/s for this feature; I'd be happy to add some more tests myself after merging.

Reply:
Done. It actually wasn't laborious at all, I just ran it with `expect(table.to_s).to eq ''` and copied the diff from the output.